CONTRIBUTORS.md

```diff
@@ -438,6 +438,8 @@ GitHub contributors:
 - Lucas CERDAN
 - Lucas Rolff
 - LucasC
+- Lukas
+- Lukas Laurinaitis
```

Comment:
Not the same person?

Reply:
regarding the emails, I have no clue it's the same person :D

Reply:
it looks like he used is personal and his professional email